cmd/swarmctl/node/common.go

```diff
@@ -138,6 +138,16 @@ func getNode(ctx context.Context, c api.ControlClient, input string) (*api.Node,
 		}
 
 		if l := len(rl.Nodes); l > 1 {
+			for _, n := range rl.Nodes {
+				name := n.Spec.Annotations.Name
```

Comment:
For now, both Docker client and swarmctl works without problem because node.Spec.Annotations.Name is never set to a valid name string. I don't know whether this will change in future. If yes, then swarmctl's way would be more preferable.